module/purpose/PS3Walk/src/PS3Walk.cpp

```diff
@@ -33,32 +34,46 @@
 #include "message/strategy/StandStill.hpp"
 
 #include "utility/input/LimbID.hpp"
+#include "utility/skill/Script.hpp"
 
 namespace module::purpose {
 
+
     using extension::Configuration;
+    using extension::behaviour::Task;
+    using message::actuation::LimbsSequence;
     using message::behaviour::state::Stability;
     using message::behaviour::state::WalkState;
     using message::skill::Kick;
     using message::skill::Look;
     using message::skill::Walk;
     using message::strategy::FallRecovery;
     using message::strategy::StandStill;
+    using utility::skill::load_script;
+
+    using extension::Configuration;
```

Comment:
Remove duplicate using and group using together by `extension`, `message` and `utility`